src/mongocxx/result/bulk_write.hpp

```diff
@@ -71,18 +74,13 @@ class MONGOCXX_API bulk_write {
     std::int32_t upserted_count() const;
 
     ///
-    /// Gets the ids of the inserted documents.
-    ///
-    /// @return The values of the _id field for inserted documents.
-    ///
-    bsoncxx::document::element inserted_ids() const;
-
-    ///
     /// Gets the ids of the upserted documents.
     ///
-    /// @return The values of the _id field for upserted documents.
+    /// @return A map from bulk write index to _id field for upserted documents.
+    ///
+    /// The returned id_map must not be accessed after the bulk_write object is destroyed.
```

Comment:
nit, just move this comment to above the return value comment, and label with @note